jobs/FreeBSD-doc-main/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
 cd doc
-make -j 2
+make HUGO_ARGS="--verbose --debug --path-warnings"
```

Comment:
Is this losing the -j 2 on purpose?

Reply:
yes, to see better the logs

Reply:
OK. As long as it's on purpose, this is a good change.

Reply:
thanks